docs/src/main/asciidoc/spring-integration-storage.adoc

```diff
@@ -95,6 +95,6 @@ public MessageHandler outboundChannelAdapter(Storage gcs) {
 }
 ----
 
-=== Sample
+==== Samples
```

Comment:
I think I need a screenshot to see what effect this has. Also, why make it plural for a single sample?

Reply:
Done and done (see below).